docs/src/main/sphinx/installation/cli.rst

```diff
@@ -405,6 +405,34 @@ Press :kbd:`→` to accept the suggestion and replace the current command line
 buffer. Press :kbd:`Ctrl+→` (:kbd:`Option+→` on Mac) to accept only the next
 keyword. Continue typing to reject the suggestion.
 
+Configuration file
+------------------
+
+The CLI can read default values for all options from a file. It uses the first
+file found from the ordered list of locations:
+
+* File path set as value of the ``TRINO_CONFIG`` environment variable.
+* ``.trino_config`` in the current users home directory.
+* ``$XDG_CONFIG_HOME/trino/config``.
+
+For example, to run the CLI with an alternative config in the current directory:
+
+.. code-block:: text
+
+    TRINO_CONFIG=cli.properties ./trino
```

Comment:
`TRINO_CONFIG=cli.properties ./trino`

@nineinchnick This line looks confusing because there is space between the filename `cli.properties` and directory (if it is directory) `./trino`. What does this command explain? 
1. Does it demonstrate an example of how to set the ENV variable and execute the trino command in one line 
or 

2. It is setting multiple values `cli. properties` and the directory `./trino` to environment variable `TRINO_CONFIG`?

The more easy-to-understand example would be where we can show users that we can configure different properties files for different authentication methods such as `kerberos-cli.properties` and `ldap-cli.properties` and then also explain what if all three files are present `kerberos-cli.properties`, `ldap-cli.properties` and `$HOME/.trino_config`

Would like to hear your and @mosabua's thought as well